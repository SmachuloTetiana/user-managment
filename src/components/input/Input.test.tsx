import React, { ChangeEvent } from "react";
import { shallow } from "enzyme";
import InputComponent from "./Input";

const props = {
    label: "Test",
    name: "text",
    type: "text",
    value: "text",
    handleInput: () => {}
}

describe("Input component", () => {
    test("should render Input component with props", () => {
        const component = shallow(<InputComponent {...props} />);

        expect(component).toMatchSnapshot()
    });
})