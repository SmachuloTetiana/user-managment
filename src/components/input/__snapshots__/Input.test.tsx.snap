// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should render Input component with props 1`] = `ShallowWrapper {}`;
